ov/storage.go

```diff
@@ -80,7 +82,7 @@ type Properites struct {
 type Volume struct {
 	InitialScopeUris utils.Nstring `json:"initialScopeUris,omitempty"` //Initial scopes for the volume.
 	IsPermanent      *bool         `json:"isPermanent,omitempty"`      //If true, indicates that the volume will persist when the profile using this volume is deleted.
-	Properties       *Properties   `json:"properties"`                 //The properties specific to a storage system family required for the creation of a storage volume.
+	Properties       *PropertiesSP `json:"properties"`                 //The properties specific to a storage system family required for the creation of a storage volume.
```

Comment:
I think we can use properties from volume resources, since both are same. 
No need to create Properties Separately for SP

Reply:
I do not want the SP to be depended on storage volume resources as it is mentioned separately in API docs also and it does relate to the storage volume properties in the API docs